lib/ansible/module_utils/scaleway.py

```diff
@@ -9,11 +9,14 @@
 
 def scaleway_argument_spec():
     return dict(
-        api_token=dict(required=True, fallback=(env_fallback, ['SCW_TOKEN', 'SCW_API_KEY', 'SCW_OAUTH_TOKEN', 'SCW_API_TOKEN']),
+        api_token=dict(required=True,
+                       fallback=(env_fallback, ['SCW_TOKEN', 'SCW_API_KEY', 'SCW_OAUTH_TOKEN', 'SCW_API_TOKEN']),
                        no_log=True, aliases=['oauth_token']),
-        api_url=dict(fallback=(env_fallback, ['SCW_API_URL']), default='https://api.scaleway.com', aliases=['base_url']),
+        api_url=dict(fallback=(env_fallback, ['SCW_API_URL']), default='https://api.scaleway.com',
+                     aliases=['base_url']),
         api_timeout=dict(type='int', default=30, aliases=['timeout']),
         query_parameters=dict(type='dict', default={}),
+        api_version=dict(default="v1"),
```

Comment:
You should always have `type` in the `dict`. While Ansible defaults to `str`, it is not good style to leave it away.

Reply:
(Also for all other `argument_spec` related stuff.)